keyboards/moky/moky88/readme.md

```diff
@@ -0,0 +1,17 @@
+# moky88
+
+An in-switch RGB LED keyboard with RGB underglow.
+
+* Keyboard Maintainer: [Vertor](https://github.com/VertorWang)
+* Hardware Supported: moky88
+* Hardware Availability: Private GB 
+
+Make example for this keyboard (after setting up your build environment):
+
+    make moky/moky88:default
+
+Flashing example for this keyboard:
+
+    make moky/moky88:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Missing bootloader section per [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md)

Reply:
Add a key to enter the bootloader, and add instructions to the readme documentation